src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/FormContext.cs

```diff
@@ -49,8 +80,23 @@ public IList<IHtmlContent> EndOfFormContent
             }
         }
 
+        /// <summary>
+        /// Gets or sets an indication the current &lt;form&gt; is associated with a tag helper or will be generated by
+        /// an HTML helper. In particular, <see cref="EndOfFormContent"/> will be rendered just prior to the next
```

Comment:
should `just prior to next &lt;/form&gt; end tag` 
be 
`just prior to &lt;/form&gt; end tag` ?